carsFilter/src/main/java/tkhal.filter.controller/FilterKafkaProducer.java

```diff
@@ -0,0 +1,46 @@
+package tkhal.filter.controller;
+
+import model.Car;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class FilterKafkaProducer {
+    private String topicFilteredName;
+    private final static Logger LOGGER = LoggerFactory.getLogger(FilterKafkaProducer.class);
+    private Producer <String, Car> producer;
+
+    public Producer <String, Car> createProducer() {
+        Properties propsRedirect = new Properties();
+
+        //reading Kafka producer properties from config file
+        try {
+            ClassLoader classloader = Thread.currentThread().getContextClassLoader();
```

Comment:
Duplicated piece of code: see `FilterKafkaConsumer` lines 22-25 and this class lines 24-27. Refactor and extract common piece

Reply:
done